src/encord_active/app/common/components/data_quality_summary.py

```diff
@@ -0,0 +1,23 @@
+from typing import Union
+
+import streamlit as st
+
+
+def summary_item(
+    section: st._DeltaGenerator,
```

Comment:
to increase reusability, should we remove the section and just return a string or an `st.markdown`?

(in the current way I would be able to render this item without a container)

Reply:
Ok. Now, summary_item only returns html string.